### ISSUE TITLE
Playground bug "undefined method '<' for Bool"

### ISSUE THREAD

**Issue by decademoon** [crystal-lang/crystal#4940]
## Crystal version

Crystal 0.23.1 (2017-07-13) LLVM 4.0.1
macOS 10.12.6
Installed with homebrew

## Code

``` cr
1 < 2 < 3
```

## Exception

```
Error in line 1: instantiating 'Crystal::Playground::Agent#i(Int32)'
in line 1: instantiating 'Crystal::Playground::Agent#i(Int32)'
in line 2: undefined method '<' for Bool
```

**Comment by straight-shoota** [comment]
Looks like #4663 which should be resolved with #4708. It works on master.

**Comment by decademoon** [comment]
Ah my mistake, I wasn't thorough enough when I searched for the issue initially.
